jetty-util/src/main/java/org/eclipse/jetty/util/Callback.java

```diff
@@ -113,13 +113,26 @@ public InvocationType getInvocationType()
     }
 
     /**
-     * Create a callback from the passed success and failure
+     * Creates a callback from the given success and failure lambdas.
```

Comment:
Not a comment for this line.... but I noticed that we have a package scope `InvocableCallback` interface that is only used once and is not necessary because all `Callback`s are `Invocable`.    Can we clean that up in this PR as well?

Reply:
fixed in #6658